server/data/collections/dataset_collection_local.yaml

```diff
@@ -0,0 +1,33 @@
+datasets:
+- dataset_name: "IRIS"
+  database_type: "PATH_DB"
+  dataset_path: "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv"
+  metadata:
+    database_type: "PATH_DB"
+    metadata_path: "../data/collections/metadata/iris_metadata.yaml"
+- dataset_name: "PENGUIN"
+  database_type: "PATH_DB"
+  dataset_path: "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv"
+  metadata:
+    database_type: "PATH_DB"
+    metadata_path: "../data/collections/metadata/penguin_metadata.yaml"
+- dataset_name: "TITANIC"
+  database_type: "S3_DB"
+  s3_bucket: "example"
+  s3_key: "data/titanic.csv"
+  endpoint_url: "http://minio:9000"
```

Comment:
The address is still a minio deployment on onyxia for the dataset_collection.yaml file, should we simply remove it completely?

Reply:
yes, if everything can be independant of onyxia, it is better

Reply:
but why is "http://minio:9000" a related to onyxia in particular ?

Reply:
It's actually not in this file. I copied the original dataset_collection.yaml into this one. In the original one, the address is http://api-lomas-mino.lab.sspcloud.fr.

But I will remove everything and also duplicated the user_collection file, so that if a user has access to something that something is indeed reachable by the server.